<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ip_cam1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>http://admin:MjcwNDE5ODcxMTE=@192.168.1.241:8080/stream/getvideo?.mjpeg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_cam1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cam1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_cam2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>rtsp://admin:admin@192.168.1.242:554/live/sub</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_cam2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cam2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start 3rd thread</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
